### PR TITLE
[DEV APPROVED] TP: 7710, Comment: Optimises Social/Corporate section

### PR DESCRIPTION
**Pull request by davidtrussler**
The purpose of this is to optimise the footer on smaller devices to avoid the need for the user to scroll too far.

The main part of this is in PR#1583 on Frontend (https://github.com/moneyadviceservice/frontend/pull/1583)

### CODE REVIEW

File: assets/layout/common/_footer.scss [hunk 1]
```diff
@@ -136,7 +139,12 @@
   .youtube-link:active,
   .youtube-link:focus {
     background: $color-youtube;
-    padding: 18.5px 14.704px; // (55px - 18px) / 2 || (55px - (213.264px * (18 / 150))) / 2
+    padding: (($icon-size - 18px) / 2) (($icon-size - (213.264px * (18 / 150))) / 2);
```
Comment (slaywell) [inline assets/layout/common/_footer.scss]: What is this doing? It's really hard to understand! I think a variable here (or just a px value) would be better than this complexity.

Comment (davidtrussler) [inline assets/layout/common/_footer.scss]: But no fun though! Actually I don't think that was supposed to make it into the final code: I was expressing it like that until we were happy with the size of the icons.

Comment (davidtrussler) [inline assets/layout/common/_footer.scss]: I've updated these values now.

**Comment by rnalexander** [comment]
:+1:

**Comment by slaywell** [comment]
Looks good. I've just realised we'll need to do a follow-up check in RAD and Universal Credit (I think these are the only ones that share Yeast) that their footers still work with these changes.
Just one question on the PR.

**Comment by slaywell** [comment]
👍

**Pull request merged**